pkg/ebpf/tracer.go

```diff
@@ -270,7 +270,7 @@ func (t *Tracer) shouldSkipConnection(conn *ConnectionStats) bool {
 	isDNSConnection := conn.DPort == 53 || conn.SPort == 53
 	if !t.config.CollectLocalDNS && isDNSConnection && conn.Direction == LOCAL {
 		return true
-	} else if util.IsBlacklistedConnection(t.sourceExcludes, conn.Source, conn.SPort) || util.IsBlacklistedConnection(t.destExcludes, conn.Dest, conn.DPort) {
+	} else if util.IsBlacklistedConnection(t.sourceExcludes, conn.Source, conn.SPort, conn.Type.String()) || util.IsBlacklistedConnection(t.destExcludes, conn.Dest, conn.DPort, conn.Type.String()) {
```

Comment:
Can we pass this as a ConnectionType instead of turning it into a string? That way we can use the type of the parameter to know what it's being used for.

Reply:
I think it would even be better to have a function accepting only a connection argument.

Reply:
humm, I guess that would result in a cyclical dependency `util <=> ebpf`. Maybe it make sense to bring this function to `ebpf`?